controlplane/src/core/services/Keycloak.ts

```diff
@@ -115,6 +115,104 @@ export default class Keycloak {
     });
   }
 
+  public async getKeycloakUsers({ realm, orgSlug }: { realm?: string; orgSlug: string }) {
```

Comment:
I'd be more explicit with the naming. Isn't it more like `getIdpUsers` ?

Reply:
no, it is to get the users from keycloak itself